### PR TITLE
feat: add hebrew locale support

### PR DESCRIPTION
**Pull request by a5r0n**

### CODE REVIEW

File: core/src/locale/he.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import type { Localization } from './types'
+
+const locale: Localization = {
+  '*': {
+    prefix: 'בכל',
+    suffix: '',
+    text: 'לא ידוע',
+    '*': {
+      empty: { text: 'בכל {{field.id}}' },
```
Comment (abichinger) [inline core/src/locale/he.ts:9]: `{{field.id}}` should only be used in the english translation.
image

For example, to translate `every month` you can set:
```ts
{
  '*': {
    month: {
      empty: { text: 'your translation' }
    }
  }
}
```

**Review by abichinger — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Why is the order of the fields different?
image

Please make sure to follow the [contribution guidlines](https://github.com/abichinger/vue-js-cron?tab=readme-ov-file#contributing)

**Comment by a5r0n** [comment]
Maybe it is an RTL issue, I tried to build and run the demo app, but no success 😕

**Comment by a5r0n** [comment]
@abichinger about the RTL issue, if you can point me with direction how to add rtl support  i will be happy to implemnt this in another PR

**Comment by abichinger** [comment]
@a5r0n I have no experience with RTL, so I don't know where to start 🤔

**Comment by a5r0n** [comment]
All we need is to add `dir="RTL"`.
I will try to implement something, should I open new PR for this?

**Comment by abichinger** [comment]
I think no changes are necessary. I just tried to set `dir="rtl"` on the `cron-light` component and it seems to work.
```html
<cron-light
      v-model="value"
      v-model:period="period"
      format="quartz"
      locale="he"
      @error="error = $event"
      dir="rtl"
    >
    </cron-light>
```
Result:
image

Is this as it should be?

**Review by abichinger — APPROVED** [review]

**Comment by a5r0n** [comment]
Yes, this is great.
I was thinking maybe we should add this as a per language settings, but maybe it's out of the scope

**Comment by abichinger** [comment]
> I was thinking maybe we should add this as a per language settings, but maybe it's out of the scope
I think this is out of scope, because of what I have read so far, the `dir` attribute is usually set on the `html` tag.

**Comment by abichinger** [comment]
Thanks again!

**Pull request merged**